nipype/interfaces/base.py

```diff
@@ -1513,13 +1359,11 @@ def _process(drain=0):
             else:
                 for stream in res[0]:
                     stream.read(drain)
+
         while proc.returncode is None:
-            if runtime_profile:
-                mem_mb, num_threads = \
-                    get_max_resources_used(proc.pid, mem_mb, num_threads)
             proc.poll()
             _process()
-            time.sleep(interval)
```

Comment:
This loop seems kind of busy. Maybe add `time.sleep(0)` to at least yield control? Or is there something in `_process()` that will yield?

Reply:
I wanted to bring it back to the original https://github.com/nipy/nipype/blame/c0d148aec7505e9c0df439c5b53d452ae478f352/nipype/interfaces/base.py#L1259 but probably @satra can enlighten us a bit here. I'd say the `time.sleep(0)` seems like a good idea.